icu4c/source/i18n/plurrule.cpp

```diff
@@ -1881,8 +1881,8 @@ void FixedDecimal::adjustForMinFractionDigits(int32_t minFractionDigits) {
 
 double FixedDecimal::getPluralOperand(PluralOperand operand) const {
     switch(operand) {
-        case PLURAL_OPERAND_N: return source;
-        case PLURAL_OPERAND_I: return static_cast<double>(intValue);
+        case PLURAL_OPERAND_N: return (exponent == 0 ? source : source * pow(10, exponent));
```

Comment:
(as stated in chat)
I think that the correct place to multiply by pow(10) is in the parsing function, not in the getPluralOperand function.  I would be much more confident in this PR if the multiplication were confined to the constructors.

Reply:
I chatted with Elango and he will prioritize https://unicode-org.atlassian.net/browse/ICU-21322 for ICU 71, and in the mean time this PR is fine for unblocking ICU 70.